config/parameters.yaml

```diff
@@ -0,0 +1,4 @@
+# Put parameters here that don't need to change on each machine where the app is deployed
+# https://symfony.com/doc/current/best_practices/configuration.html#application-related-configuration
```

Comment:
Won't this comment disappear immediately after running the install command?
Maybe we should just remove it.

Reply:
Yeah it disappears